### PR TITLE
[zh]Update host in example-ingress.yaml to hello-world.example

### PR DESCRIPTION
**Pull request by ydFu**
The file `content/en/docs/tasks/access-application-cluster/ingress-minikube.md` has been updated in this PR (https://github.com/kubernetes/website/pull/46239) ,so we will synchronize the localization.

### CODE REVIEW

File: content/zh-cn/docs/tasks/access-application-cluster/ingress-minikube.md [hunk 1]
```diff
@@ -276,49 +378,55 @@ The following manifest defines an Ingress that sends traffic to your Service via
    Version: 1.0.0
    Hostname: web-55b8c6998d-8k564
    ```
+<!--
+1. Optionally, you can also visit `hello-world.example` from your browser.
+-->
+5. 可选步骤：你也可以从你的浏览器中访问 `hello-world.example`。
 
+   {{< note >}}
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/access-application-cluster/ingress-minikube.md:386]: `{{< /note >}}` is missing, which causes the ci failed.

Comment (ydFu) [reply content/zh-cn/docs/tasks/access-application-cluster/ingress-minikube.md]: @windsonsea thank,fix it.

File: content/zh-cn/docs/tasks/access-application-cluster/ingress-minikube.md [hunk 2]
```diff
@@ -157,9 +189,12 @@ If you haven't already set up a cluster locally, run `minikube start` to create
    ```
 
 <!--
-1. Visit the Service via NodePort:
+1. Visit the Service via NodePort, using the [`minikube service`](https://minikube.sigs.k8s.io/docs/handbook/accessing/#using-minikube-service-with-tunnel) command. Follow the instructions for your platform:
 -->
-4. 使用节点端口信息访问服务：
+4. 通过 NodePort 访问 Service，使用 [`minikube service`](https://minikube.sigs.k8s.io/docs/handbook/accessing/#using-minikube-service-with-tunnel) 命令。根据你的平台执行以下操作：
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/access-application-cluster/ingress-minikube.md:194]:
```suggestion
4. 使用 [`minikube service`](https://minikube.sigs.k8s.io/docs/handbook/accessing/#using-minikube-service-with-tunnel) 命令通过
   NodePort 访问 Service。根据你的平台执行以下操作：
```

File: content/zh-cn/docs/tasks/access-application-cluster/ingress-minikube.md [hunk 3]
```diff
@@ -173,11 +208,37 @@ If you haven't already set up a cluster locally, run `minikube start` to create
...
+   -->
+   使用上一步输出的 URL：
    ```shell
    curl http://172.17.0.15:31637
    ```
-
+   {{% /tab %}}
+   {{% tab name="MacOS" %}}
+   ```shell
+   # The command must be run in a separate terminal.
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/access-application-cluster/ingress-minikube.md:221]:
```suggestion
   # 这条命令必须在一个独立的终端中运行
```

File: content/zh-cn/docs/tasks/access-application-cluster/ingress-minikube.md [hunk 4]
```diff
@@ -253,19 +314,60 @@ The following manifest defines an Ingress that sends traffic to your Service via
...
+1. Verify that the Ingress controller is directing traffic, by following the instructions for your platform:
 -->
-4. 验证 Ingress 控制器能够转发请求流量：
+4. 验证 Ingress 控制器是否正在转发流量，根据你的平台执行以下操作：
+   {{< note >}}
+   <!--
+   The network is limited if using the Docker driver on MacOS (Darwin) and the Node IP is not reachable directly. To get ingress to work you’ll need to open a new terminal and run `minikube tunnel`.  
+   `sudo` permission is required for it, so provide the password when prompted. 
+   -->
+   如果在 MacOS（Darwin）上使用 Docker 驱动程序，网络是受限的，而且节点 IP 无法直接访问。要使 Ingress 正常工作，你需要打开一个新的终端并运行 `minikube tunnel`。
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/access-application-cluster/ingress-minikube.md:330]:
```suggestion
   如果在 MacOS（Darwin）上使用 Docker 驱动程序，网络是受限的，而且节点 IP 无法直接访问。
   要使 Ingress 正常工作，你需要打开一个新的终端并运行 `minikube tunnel`。
```

File: content/zh-cn/docs/tasks/access-application-cluster/ingress-minikube.md [hunk 5]
```diff
@@ -276,49 +378,54 @@ The following manifest defines an Ingress that sends traffic to your Service via
...
-     ```
+     {{< tabs name="hosts" >}}
+     {{% tab name="Linux" %}}
+   <!--
+   Look up the external IP address as reported by minikube
+   -->
+   查找 minikube 报告的外部 IP 地址：
+   ```shell
+   ```none
+     minikube ip
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/access-application-cluster/ingress-minikube.md:400]: The preview is not good here: https://deploy-preview-46261--kubernetes-io-main-staging.netlify.app/zh-cn/docs/tasks/access-application-cluster/ingress-minikube/

**Review by windsonsea — COMMENTED** [review]
See preview: https://deploy-preview-46261--kubernetes-io-main-staging.netlify.app/zh-cn/docs/tasks/access-application-cluster/ingress-minikube/

File: content/zh-cn/docs/tasks/access-application-cluster/ingress-minikube.md
Comment (mengjiao-liu) [inline content/zh-cn/docs/tasks/access-application-cluster/ingress-minikube.md:1]: PR https://github.com/kubernetes/website/pull/46250 has synchronized the file.

**Comment by windsonsea** [comment]
oops, need to resolve the conflicts.

/approve cancel

**Comment by ydFu** [comment]
PR https://github.com/kubernetes/website/pull/46250 has synchronized the file `content/en/docs/tasks/access-application-cluster/ingress-minikube.md`.
Update `content/zh-cn/examples/service/networking/example-ingress.yaml` from this PR (https://github.com/kubernetes/website/pull/46239)

/cc @mengjiao-liu @windsonsea

**Pull request merged**